### PR TITLE
[Wasm64] Fix autoconf tests, and enable test_bullet

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: emcc.py [hunk 1]
```diff
@@ -630,10 +630,15 @@ def check_human_readable_list(items):
 def make_js_executable(script):
   src = read_file(script)
   cmd = config.JS_ENGINE
-  if settings.WASM_BIGINT:
+  if settings.MEMORY64 == 1:
+    if not config.V8_ENGINE:
+      exit_with_error('V8_ENGINE is required in order to run configure tests with MEMORY64 enabled')
```
Comment (dschuff) [inline emcc.py:635]: Recent version of node (e.g. 18) do have `--experimental-wasm-memory64`. Should we just upgrade?

Comment (sbc100) [reply emcc.py:635]: IIRC upgrade node is a little complicated/controversial: See https://github.com/emscripten-core/emsdk/pull/829.

I mean we probably should, but its a complex enough discussion that I don't think we would want to block this change on that.

Comment (dschuff) [reply emcc.py:635]: ok yeah, we should probably have a separate discussion about node policy.

**Review by dschuff — APPROVED** [review]

File: emcc.py (hunk 1 above)
Comment (sbc100) [reply emcc.py]: Done.. switched to modern node instead.

**Review by dschuff — APPROVED** [review]

**Pull request merged**